src/main/java/com/epam/test_generator/dto/StepDTO.java

```diff
@@ -9,6 +12,9 @@
 
     private Long id;
 
+    @JsonIgnore
+    private String action;
```

Comment:
I would rather suggest using enum here to cover most of possible errors during compile time

Reply:
Done

Reply:
You see the old commit, the property has already changed to enum type